langchain/custom-tool.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const { Tool } = require('@langchain/core/tools')
+const data = {
```

Comment:
The response must be a scalar. so you should remove the inner object and just have the value of response be the value of the parent key

Reply:
OK. That's updated.